embedded/store/immustore.go

```diff
@@ -2737,3 +2828,163 @@ func minUint64(a, b uint64) uint64 {
 	}
 	return a
 }
+
+// readTxOffsetAt reads the value-log offset of a specific entry (index) in a transaction (txID)
+func (s *ImmuStore) readTxOffsetAt(txID uint64, allowPrecommitted bool, index int) (*TxEntry, error) {
```

Comment:
method name and index parameter are not self-explanatory

Reply:
fixed